app/src/main/res/values/strings.xml

```diff
@@ -149,4 +149,22 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="url_terms">https://anitab.org/terms-of-use/</string>
     <string name="url_privacy">https://anitab.org/privacy-policy/</string>
     <string name="url_code_of_conduct">https://ghc.anitab.org/code-of-conduct/</string>
+    <string name="Name">Name</string>
```

Comment:
Keep the first letter of the string name lowercase in all the fields that you've added - 
e.g. - `<string name="name">Name</string>`